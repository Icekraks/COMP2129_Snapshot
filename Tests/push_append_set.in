set a 1 2 3
get a
append a 4 3
get a
push 8 9
get a
bye
