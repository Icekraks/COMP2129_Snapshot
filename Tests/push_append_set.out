ok
[1 2 3]
ok
[1 2 3 4 3]
ok
[9 8 1 2 3 4 3]
bye
